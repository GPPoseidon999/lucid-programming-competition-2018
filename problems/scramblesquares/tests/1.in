3 3
0 7 7 5
0 9 0 7
5 5 1 0
1 0 0 9
7 4 5 0
0 9 4 2
0 2 4 6
1 0 0 8
5 0 5 9
