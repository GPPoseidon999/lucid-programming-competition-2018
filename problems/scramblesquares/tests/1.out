\ 1 /\ 5 /\ 0 /
8 X 00 X 55 X 7
/ 0 \/ 1 \/ 7 \
\ 0 /\ 1 /\ 7 /
7 X 99 X 00 X 4
/ 0 \/ 0 \/ 5 \
\ 0 /\ 0 /\ 5 /
6 X 22 X 99 X 0
/ 4 \/ 4 \/ 5 \
